packages/editor/src/components/prefabs/PrefabEditors.tsx

```diff
@@ -143,5 +144,20 @@ export const PrefabShelfState = defineState({
         url: `${config.client.fileServer}/projects/default-project/assets/prefabs/fog.prefab.gltf`,
         category: 'Lookdev'
       }
-    ] as PrefabShelfItem[]
+    ] as PrefabShelfItem[],
+  reactor: () => {
+    const shelfState = useHookstate(getMutableState(PrefabShelfState))
+    return shelfState.value.map((shelfItem) => <ShelfItemReactor key={shelfItem.url} url={shelfItem.url} />)
+  }
 })
+
+const ShelfItemReactor = (props: { key: string; url: string }): JSX.Element | null => {
+  useEffect(() => {
```

Comment:
We have fairly robust caching already if you change this useEffect to useGLTF(props.url)

Reply:
That's how the model component works normally and it's causing fetches every time on these prefabs

Reply:
oh weird it works. Why isn't the useGLTFDocument in the model component already caching then?

Reply:
I'd need some more context around what's happening, but my best guess is that the model components are being unmounted/remounted and if they're the only reference to the asset it will be removed from the cache and then re-added. I'm sure there's some tweaks we can do to the cache lifetimes if it's causing issues.

Reply:
oh yes the dereference is unmounting them